src/controller/genericmidi.cxx

```diff
@@ -623,28 +398,30 @@ void GenericMIDI::launchScene( int scene )
     
     if ( b->action == GRID_LAUNCH_SCENE )
     {
-      for( int i = 0; i < 5; i++ )
```

Comment:
Hmm, I'm not totally convinced just yet, but I also don't have hardware to test. I'll have to review this next week with access to hardware.